samples/quickstart.js

```diff
@@ -13,31 +13,25 @@
 
 'use strict';
 
-async function main() {
+async function main(projectId) {
   // [START nodejs_appengine_admin_quickstart]
   // Imports the Google Cloud client library
 
-  // remove this line after package is released
-  // eslint-disable-next-line node/no-missing-require
-  const {VersionsClient} = require('@google-cloud/appengine-admin');
+  const {ServicesClient} = require('@google-cloud/appengine-admin');
 
   // TODO(developer): replace with your prefered project ID.
   // const projectId = 'my-project'
 
   // Creates a client
-  // eslint-disable-next-line no-unused-vars
-  const client = new {VersionsClient}();
+  const client = new ServicesClient();
 
-  //TODO(library generator): write the actual function you will be testing
-  async function doSomething() {
-    console.log(
-      'DPE! Change this code so that it shows how to use the library! See comments below on structure.'
-    );
-    // const [thing] = await client.methodName({
-    // });
-    // console.info(thing);
+  async function listVersions() {
+    const [versions] = await client.listServices({
+      parent: `apps/${projectId}`,
+    });
+    console.info(versions);
   }
-  doSomething();
+  listVersions();
   // [END nodejs_appengine_admin_quickstart]
```

Comment:
Are we sure we have this snippet region tag right?  I didn't think it used language like this, and I am not sure what product name it will expect (peeks at snippet bot)

Reply:
This region tag doesn't currently link to any samples on cloud.google.com. Right now it's more of a placeholder in case the team were to update the samples, so they would know to add a region tag and appropriately register it/etc. 

Reading up on region tags though, it probably would be more appropriate to call it `appengine_quickstart`. Do you think it's worth updating considering this doesn't really point to anything today?

Reply:
Yeah let's keep these consistent.  @tmatsuo I'm a little surprised snippet bot didn't yell louder at me :)